Src/BootCamp.Chapter/AsymetricalArmor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class AsymetricalArmor : Armor
```

Comment:
While being possible in the current scope this still is nothing I would recommend. You do not need the extra logic at all.

Reply:
The same logic used here would just be repeated in Armpiece and Shoulder piece class. There is no extra logic being added, just making a base class instead of having the same code copied all over the solution.

Reply:
But you do not use it once. `GetIsLeft()` was never used. Your justification would be fine if you make use of your logic you coded in, but you didn't. Because of that you made something more complex than needed. Also you did not make real use of any constructor that needs isLeft. Actually it doesn't matter if you set the "same" Armor to the left and the right or two different ones.

Reply:
I might have misunderstood the task. Do Armpiece and Shoulderpiece not have a specified side, and instead the player decides if to use one of those on the left or the right?

Reply:
Well I removed them now, gonna leave the conversation open for now just in case.

Reply:
This is rather a UI-interaction problem than a problem that should be solved through inheritance IMO. It could be possible to solve that through it, though. However, your code did lack the additional logic needed to justify the left and right side differenciation. That is the main point. Your old version didn't fail on trying to put a right side item on the left for example. :)